src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -109,35 +112,35 @@
       SelectionChanged="Versions_SelectionChanged"
       SelectedItem="{Binding Path=SelectedVersion, Mode=TwoWay}">
       <ComboBox.ItemContainerStyle>
-            <Style TargetType="{x:Type ComboBoxItem}" BasedOn="{StaticResource {x:Type ComboBoxItem}}">
-                <Setter Property="Foreground" Value="{DynamicResource {x:Static local:Brushes.UIText}}" />
-                <Style.Triggers>
-                    <DataTrigger Binding="{Binding Path=IsValidVersion}" Value="false">
-                        <Setter Property="Foreground" Value="{x:Static SystemColors.GrayTextBrush}" />
-                    </DataTrigger>
-                    <DataTrigger
+        <Style TargetType="{x:Type ComboBoxItem}" BasedOn="{StaticResource {x:Type ComboBoxItem}}">
+          <Setter Property="Foreground" Value="{DynamicResource {x:Static local:Brushes.UIText}}" />
+          <Style.Triggers>
+            <DataTrigger Binding="{Binding Path=IsValidVersion}" Value="false">
+              <Setter Property="Foreground" Value="{x:Static SystemColors.GrayTextBrush}" />
+            </DataTrigger>
+            <DataTrigger
                         Binding="{Binding Converter={StaticResource NotNullToBooleanConverter}}"
                         Value="True" >
-                        <Setter
+              <Setter
                             Property="IsEnabled"
                             Value="{Binding IsValidVersion}" />
-                        <Setter
+              <Setter
                             Property="ToolTipService.ShowOnDisabled"
                             Value="True" />
-                        <Setter
+              <Setter
                             Property="ToolTip"
                             Value="{Binding Converter={StaticResource TooltipConverter}}" />
```

Comment:
With the update to the indentation level of the Setter tags, their properties are now too far indented.